packages/runner/src/builder/opaque-ref.ts

```diff
@@ -19,6 +19,7 @@ import { getTopFrame, recipe } from "./recipe.ts";
 import { createNodeFactory } from "./module.ts";
 
 let mapFactory: NodeFactory<any, any>;
+let mapWithPatternFactory: NodeFactory<any, any>;
```

Comment:
* Shouldn't this be `NodeFactory<any, any> | undefined`?
* Could `NodeFactory`s generics be `unknown`? (maybe not, could be too difficult)